backend/monitoring/dashboard.py

```
@@ -707,6 +732,343 @@ async def trigger_test(test_type: str, admin=Depends(get_current_admin_dependenc
+            data=result
+        )
+    except Exception as e:
+        logger.error(f"Test execution failed: {e}")
+        return StandardResponse(
+            status="error",
+            message=f"Failed to execute test: {str(e)}",
+            data={}
+        )
+
+@router.get("/test-suites")
+async def get_available_test_suites(admin=Depends(get_current_admin_dependency)):
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)